wire-ios-sync-engine/Source/SessionManager/SessionFactories.swift

```diff
@@ -82,6 +83,7 @@ open class AuthenticatedSessionFactory {
             application: application,
             cryptoboxMigrationManager: CryptoboxMigrationManager(),
             coreDataStack: coreDataStack,
+            analyticsSessionConfiguration: analyticsSessionConfiguration,
```

Comment:
Is this still needed?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e12bacb446dc277c68c1c3efba1ebb121183ea68